### DISCUSSION TITLE
[Config help] Renovate CLI GPG key

### DISCUSSION THREAD

**Discussion by k4y4k** [renovatebot/renovate#9368] [Request Help] [answered]
I have a few private repos here that I use the renovate cli from npm for. Currently I feed that CLI an access token from my -- that is, this same -- github account, with repo scope.

However, commits show up as [Unverified]. I'm willing to jump through hoops galore to get a [Verified] GPG signature for commits to a private repo on github.com, but the docs are kind of vague on this. Any pointers?

**Comment by rarkins** [comment, marked as answer]
The docs are brief because it shouldn't be hard, see https://docs.renovatebot.com/self-hosted-configuration/#gitprivatekey

If you want to explain what you tried or what you don't understand, then we have a starting point.
